proto/src/rr/dnssec/rdata/nsec3.rs

```diff
@@ -348,14 +359,22 @@ pub fn decode_type_bit_maps(
                 // window is the Window Block # from above
                 // len is the Bitmap Length
                 // current_byte is the Bitmap
-                let mut bit_map = current_byte;
+                let mut bit_map = current_byte.unverified(/*validated and restricted in usage in following usage*/);
 
                 // for all the bits in the current_byte
                 for i in 0..8 {
                     // if the current_bytes most significant bit is set
                     if bit_map & 0b1000_0000 == 0b1000_0000 {
                         // len - left is the block in the bitmap, times 8 for the bits, + the bit in the current_byte
-                        let low_byte = ((len - left) * 8) + i;
+                        // let block = len
```

Comment:
Forgotten block of commented out code here?

Reply:
Oops. Will fix.